packages/mergebot/src/basic.ts

```diff
@@ -9,7 +9,21 @@ export type ColumnName =
   | BlessedColumnName
   | "*REMOVE*"; // special value: indicates closing the PR
 
-export type BlessedColumnName = "Waiting for Code Reviews (Blessed)";
+export type BlessedColumnName = (typeof blessedColumnNames)[number];
+
+const blessedColumnNames = [
+  "Waiting for Code Reviews (Blessed)",
+  "Waiting for Author to Merge (Blessed)",
```

Comment:
does the weekly maintainer have to know which (Blessed) column to move to?

Reply:
No, they should only ever put something into the first one, since I removed the "straight to mergable" thing.